src/inventorymanager.cpp

```diff
@@ -503,8 +516,7 @@ void IMoveAction::apply(InventoryManager *mgr, ServerActiveObject *player, IGame
 			<< std::endl;
 
 	// If we are inside the move somewhere loop, we don't need to report
-	// anything if nothing happened (perhaps we don't need to report
-	// anything for caused_by_move_somewhere == true, but this way its safer)
+	// anything if nothing happened
```

Comment:
We do need to report for `caused_by_move_somewhere == true`, as the callbacks take the inventory slots.